examples/java/spring-boot/build.gradle

```
@@ -12,6 +12,8 @@ apply plugin: 'jsonschema2pojo'
 
 dependencies {
   implementation 'org.springframework.boot:spring-boot-starter-web'
+  implementation 'org.springframework.boot:spring-boot-starter-log4j2:2.6.1'
+  implementation 'org.apache.logging.log4j:log4j-layout-template-json:2.16.0'
```

These versions are provided by the Spring Boot dependencies plugin, so we can change it to:

```suggestion
  implementation 'org.springframework.boot:spring-boot-starter-log4j2'
  implementation 'org.apache.logging.log4j:log4j-layout-template-json'
```